uuid/uuid_fsm.rl

```diff
@@ -41,7 +41,7 @@ func matchUuidRegexAndroid(data string) bool {
 	cs, p, pe := 0, 0, len(data)
 	eof := pe
 	%%{
-		main := [a-f0-9]{8} '-' [a-f0-9]{4} '-' [a-f0-9]{4} '-' [a-f0-9]{4} '-' [a-f0-9]{12} %{ return true } ;
+		main := [a-f0-9]{8} '-' [a-f0-9]{4} '-' [a-f0-9]{4} '-' [8|9|a|b] [a-f0-9]{3} '-' [a-f0-9]{12} %{ return true } ;
```

Comment:
so this is what is the real UUID now.
We need to decide what to do

* Change the regex and therefore the logic
* Decide that these regex are fine and leave it as it is

Reply:
maybe it makes sense to bring this up with @itszootime - it's not clear to me who should be deciding that

Reply:
Let's keep the more lenient regexp from https://github.com/remerge/go-tools/pull/12 and generate the correct Go code, otherwise we have an issue where `IsValid` can be true but both `IsiOS` and `IsAndroid` are false.

Reply:
sure! So instead I changed generated files and left regexp as it was